README.md

```diff
@@ -1,291 +1,151 @@
+- **Generate Shopping list According to recipe added food**
+- **Public And Private Recipe feature**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
+> clone this repo: https://github.com/atok624/Recipe-App.git
+
 ### Prerequisites
 
-In order to run this project you need:
+In order to run this project you need ruby, rails and postgres installed:
 
-<!--
-Example command:
+On Linux:
 
 ```sh
- gem install rails
+ sudo apt-get install ruby-full
 ```
- -->
+
+On Windows
+
+Go to [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/about) to set up wsl for ubuntu
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
 ```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git https://github.com/atok624/Recipe-App.git
+  cd Recipe-App
 ```
---->
 
 ### Install
 
-Install this project with:
-
-<!--
-Example command:
-
 ```sh
-  cd my-project
-  gem install
+  bundle install
 ```
---->
 
 ### Usage
 
-To run the project, execute the following command:
-
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
-
-### Run tests
+**To run the project follow the following instruction:-**
 
-To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+- Run **rails server**
 
 ### Deployment
 
-You can deploy this project using:
-
-<!--
-Example:
-
-```sh
-
-```
- -->
+You can deploy this project using: **Not available**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
-
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+👤 **Nicholas Kwamena Amissah** <a name="authors"></a>
 
-<!-- FUTURE FEATURES -->
+- GitHub: [Nicholas Amissah](https://github.com/atok624)
+- Twitter: [Nicholas Amissah](https://twitter.com/MysticalAmissah)
+- LinkedIn: [Nicholas Amissah](https://www.linkedin.com/in/nicholas-amissah-153b09154)
 
-## 🔭 Future Features <a name="future-features"></a>
+👤 **Gilbert Mutai**
 
-> Describe 1 - 3 features you will add to the project.
+- GitHub: [GitHub](https://github.com/Mutai-Gilbert)
+- Twitter: [Gilbert Mutai](https://twitter.com/iam_pkoech)
+- LinkedIn: [Gilbert Mutai](https://www.linkedin.com/in/mutai-kipkoech/)
 
```

Comment:
- [optional] your README file is missing the future features section. Remember, this section is an important part of the README file, as it provides the current status of the project and the future direction it aims to improve.

Reply:
Thanks so much 
We will resolve that